eventstore/tasks.py

```diff
@@ -14,6 +21,10 @@
 from ndoh_hub.utils import rapidpro
 
 
+def get_utc_now():
+    return datetime.utcnow()
```

Comment:
We should probably work with timezone aware timestamps instead, ie. `datetime.now(tz=pytz.utc)`, to avoid timezone issues when comparing timestamps